### DISCUSSION TITLE
GitVersion always pick up the highest version instead of the newly created one

### DISCUSSION THREAD

**Discussion by nathalawi** [GitTools/GitVersion#3808] [Q&A] [answered]
We were previously creating some tags but now we want to use Gitversion on Gitlab but I am facing an this issue : I want to initialize to version like 0.1.0 but the tool is not picking this tag I created but the highest one existed. Do you have any other way to override this behavior ?

**Comment by asbjornu** [comment]
Have you tried to [`ignore`](https://gitversion.net/docs/reference/configuration#ignore) the problematic tag(s)?

**Comment by nathalawi** [comment]
Thank you @asbjornu for the idea but I have like around 20 tags for one project

**Comment by asbjornu** [comment, marked as answer]
Have you at least tried the [`commits-before` configuration property](https://gitversion.net/docs/reference/configuration#commits-before)?

**Comment by nathalawi** [comment]
I have just tried this and it works. Thanks  @asbjornu 
```
ignore:                              
  sha: []                            
  commits-before: yyyy-MM-ddTHH:mm:ss
```
